File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string ineq, string eq, int n, int m) {
    eq = ineq + eq;
    if (eq == ">="){
        if(n >= m) return 1;
    } else if ("<=" == eq){
        if(n <= m) return 1;
    } else if (">!" == eq){
        if(n > m) return 1;
    } else if ("<!" == eq){
        if(n < m) return 1;
    } 
    return 0;
}

File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        eq = ineq + eq;
        if (">=".equals(eq)){
            if(n >= m) return 1;
        } else if ("<=".equals(eq)){
            if(n <= m) return 1;
        } else if (">!".equals(eq)){
            if(n > m) return 1;
        } else if ("<!".equals(eq)){
            if(n < m) return 1;
        } 
        return 0;
    }
}